### DISCUSSION TITLE
MSSQL server connector - cdc schema excluded

### DISCUSSION THREAD

**Discussion by Andrew-InTheBox** [airbytehq/airbyte#35737] [Connector Ideas and Features]
I had an edge case that led me to find this block of code in the MSSQL server connector source.  I had a use case to replicate the cdc schema for a database and found that it's being hard coded into an exclusion set.  Might be useful to make that configurable, or just remove CDC from that set of excluded schemas.  

What I'm doing is some development and testing work related to data vault design.  Normally letting airbyte handle CDC would be fine, but in this case I do actually want to replicate dbo and cdc schemas for some development.

https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlSource.java

```
@Override
public Set<String> getExcludedInternalNameSpaces() {
  return Set.of(
      "INFORMATION_SCHEMA",
      "sys",
      "spt_fallback_db",
      "spt_monitor",
      "spt_values",
      "spt_fallback_usg",
      "MSreplication_options",
      "spt_fallback_dev",
      "cdc"); // is this actually ok? what if the user wants cdc schema for some reason?
}
```

In summary, this user actually does want cdc schema for some reason.  But I recognize that's not likely very common.

[a comment by an automated account is not shown]
